src/helpers/index.tsx

```diff
@@ -19,35 +20,35 @@ export async function getMarketPrice({ networkID, provider }) {
   return marketPrice;
 }
 
-export function shorten(str) {
+export function shorten(str: string) {
   if (str.length < 10) return str;
   return `${str.slice(0, 6)}...${str.slice(str.length - 4)}`;
 }
 
-export function trim(number, precision) {
+export function trim(number: number | undefined, precision: number | undefined) {
   if (number == undefined) {
     number = 0;
   }
   const array = number.toString().split(".");
   if (array.length === 1) return number.toString();
 
-  array.push(array.pop().substring(0, precision));
+  array.push((array.pop() ?? "0").substring(0, precision));
```

Comment:
This feels bad. :(

Reply:
`array.pop()` can return undefined, I believe the desired default is 0, but is the syntax the issue?

Reply:
@lolchocotaco I modified it slightly, lmk if it is okay now.

Reply:
lgtm